org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/MonitorTest.java

```diff
@@ -361,6 +365,7 @@ public void runWillPerform(RunEvent evt) throws ScanningException{
 	}
 
 	private List<IScannable<?>> createMonitors(List<String> monitorNames) throws ScanningException {
+		if (monitorNames == null) return null; 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)